scripts/fuzz.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+function run_fuzz() {
+  PACKAGES=$1
+  RUN_TIME=$2
+  NUM_WORKERS=$3
+  TIMEOUT=$4
+
+  for pkg in $PACKAGES; do
+    pushd $pkg
+
+    go test -list=Fuzz.* | grep Fuzz | while read line; do
+        echo ----- Fuzz testing $pkg:$line for $2 with $3 workers -----
+        go test -fuzz=^$line\$ -test.timeout=$4 -fuzztime=$2 -parallel=$3
```

Comment:
Do we need to set up an Actions cache so the generated corpus goes into there?

Reply:
We don't currently run anything in CI yet, this is just for easily running things locally. I guess the question is: Do we want to use the same pattern as in `lnd` where the corpus is in a separate repo?
IMO we don't need to make it that complicated and can just commit them to this repo directly? Since everything will be below `testdata` folders anyway.

Reply:
Shellcheck mentions that all the variables in here need double quoting. And that args $2, $3, and $4 above are unused.

Reply:
> IMO we don't need to make it that complicated and can just commit them to this repo directly? Since everything will be below testdata folders anyway.



Np committing to the same repo for now. If the corpus gets massive, then we may want to externalize it.

Reply:
Cool, going to let them run for a full hour each and will then commit the corpus.